assets/html/documenter.css

```diff
@@ -428,3 +432,98 @@ article section.docstring a.source-link {
     background-color: transparent;
     padding: 0;
 }
+
+@media only screen and (max-width: 768px) {
+    div#topbar {
+        display: block; /* is mobile */
+        position: fixed;
+        width: 100%;
+        height: 2.7em;
+        background-color: white;
```

Comment:
I would also give the `#topbar` a slight background color as well, probably the same as the menu `#fcfcfc`.